tests/unit/test_text_chat.py

```
@@ -104,21 +85,53 @@ def create_offline_client(test_provider: str = TEST_PROVIDER, model_override: Op
         vision_model=profile.get('VISION_MODEL', ''),
         vision_base_url=profile.get('VISION_BASE_URL', ''),
         vision_api_key=profile.get('VISION_API_KEY', ''),
         on_text_delta=AsyncMock(),
         on_response_done=AsyncMock()
     )
-    return client
+
+
+# 10-round conversation prompts — designed to test context retention & natural flow
+MULTI_TURN_PROMPTS = [  # noqa: RUF001
+    "你好呀！最近过得怎么样？",
```

Ruff (0.15.2)

[warning] String contains ambiguous `！` (FULLWIDTH EXCLAMATION MARK). Did you mean `!` (EXCLAMATION MARK)?

(RUF001)